spec/unit/generator/xcconfig/pod_xcconfig_spec.rb

```diff
@@ -165,6 +165,13 @@ module XCConfig
             xcconfig.to_hash['OTHER_LDFLAGS'].should == '-ObjC -framework "CoconutLib"'
           end
 
+          it 'includes other ld flags for transitive dependent targets' do
```

Comment:
This test covers the case where vendored artifacts are now properly link to test xcconfigs.